charts/jellyseerr-chart/values.yaml

```diff
@@ -16,6 +16,25 @@ fullnameOverride: ""
 strategy:
   type: Recreate
 
+# Liveness / Readiness / Startup Probes
+probes:
+  # -- Configure liveness probe (set to null to disable)
```

Comment:
Doing so without proper context seems like a really bad practice, you can edit the configuration to fit your needs.

Reply:
Hi, I don't understand your point, doing what ? Disabling probe ? If so it's indeed a bad practice. But I think that it's important to keep users the possibility to explicitly do unclean setups. By default probes remains

Reply:
Yes, allowing the probe to be disabled. I don’t see a need for it at the moment, so if you don’t need it, I’d prefer to keep it as is. If someone requests it in the future, we can discuss it then and it would not be a breaking change.